astro/src/content/blog/digital-twins.mdx

```diff
@@ -1,7 +1,8 @@
 ---
 publish_date: 2025-04-16
 title: "Digital Twins: Bridging Physical Assets and Digital Identities"
-description: Learn about digital twins—virtual replicas of physical systems—and their role in optimizing performance and reducing downtime in the real world. 
+seoTitle: "Digital Twin Technology and Identity | Applying Digital Twins Beyond Physical Assets"
+description: Explore how digital twin technology applies to identity systems, not just physical asset twins. Learn how concepts from digital twins can enhance user modeling and auth architecture.
```

Comment:
this description isn't what this blog post does at all. The whole point is that we're using FusionAuth to control what  physical assets can do. There's nothing about user modeling or auth architecture.

What am I missing?